.eslintrc.json

```diff
@@ -0,0 +1,21 @@
+{
+    "env": {
+        "browser": true,
+        "es2021": true,
+        "node": true
+    },
+    "extends": "standard",
+    "overrides": [
+    ],
+    "parserOptions": {
+        "ecmaVersion": "latest",
+        "sourceType": "module"
+    },
+    "rules": {
```

Comment:
The linter only seems to run on `main.js` and `src/config.js`. Many files in `src/` are in violation with the new linter rules, but have not been touched.

Reply:
Fair. Thanks